dotnet/src/VectorDataIntegrationTests/AzureAISearchIntegrationTests/HybridSearch/AzureAISearchKeywordVectorizedHybridSearchTests.cs

```diff
@@ -19,23 +18,19 @@ public class AzureAISearchKeywordVectorizedHybridSearchTests(
         IClassFixture<AzureAISearchKeywordVectorizedHybridSearchTests.VectorAndStringFixture>,
         IClassFixture<AzureAISearchKeywordVectorizedHybridSearchTests.MultiTextFixture>
 {
-#pragma warning disable CA1308 // Normalize strings to uppercase
-    private static readonly string _testIndexPostfix = new Regex("[^a-zA-Z0-9]").Replace(Environment.MachineName.ToLowerInvariant(), "");
-#pragma warning restore CA1308 // Normalize strings to uppercase
-
     public new class VectorAndStringFixture : KeywordVectorizedHybridSearchComplianceTests<string>.VectorAndStringFixture
     {
         public override TestStore TestStore => AzureAISearchTestStore.Instance;
 
         // Azure AI search only supports lowercase letters, digits or dashes.
-        public override string CollectionName => "vecstring-hybrid-search-" + _testIndexPostfix;
+        public override string CollectionName => "vecstring-hybrid-search-" + AzureAISearchTestEnvironment.TestIndexPostfix;
```

Comment:
At some point we should look at centralizing the test collection naming policies rather than having to do that manually for each test...